examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<blockquote>it allows users to simulate small-to-intermediate-scale circuits with significantly reduced costs.&nbsp;</blockquote><p>Suggest: "it allows users to simulate task-heavy workloads with small-to-intermediate-scale circuits with significantly reduced costs."</p><p>The idea is that Jobs is not good for one small circuit because of container time.</p><p> </p><p> </p><blockquote>Embedded simulator is a new feature released on May 4, 2022 by the Braket team.&nbsp;...</blockquote><p>I understand that this sentence is to justify "new" feature. But it a bit unnecessary. Suggest:"Embedded simulator enables Braket Jobs users to use the hosting EC2 instance ...."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<blockquote>&nbsp;... with significantly reduced costs.</blockquote><p>Just curious, is this really true? We've been focus on reducing run time when switching from on-demand to embedded sim. I didn't compare the dollar cost. Do you have any data for this?</p>